vm/systemSmartContracts/staking.go

```diff
@@ -127,6 +138,78 @@ func getPercentageOfValue(value *big.Int, percentage float64) *big.Int {
 	return result
 }
 
+func (r *stakingSC) getConfig() *StakingNodesConfig {
+	config := &StakingNodesConfig{
```

Comment:
no need for a mutex in this and the following methods?

Reply:
no need for mutex - smart contract are not called in parallel.

Reply:
ok